docs/messages/0180.md

```diff
@@ -0,0 +1,24 @@
+# Warning: 0180
+
+No pilet.json has been found.
+
+## Description
+
+For some functionality such as multi Piral instance support when debugging
+a pilet, a special file called pilet.json is required. While this file is
+optional in general, it must be available for certain tasks such as adding
+or removing a Piral instance for debugging purposes.
+
+## Example
+
+If no pilet.json is yet available you can create one. It should be adjacent to
+the package.json of your pilet, even though different locations are also possible.
+By default, the following content can be used for an empty file:
+
+```json
+
```

Comment:
I'm not sure about this comment, but here goes: Is an empty file without any content really okay in this scenario or is the file missing some content like `{ }` for example?
Just ignore this if everything's as intended here.. 😉

Reply:
Fixed!